CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # SpectralOps - Automated Code Security Change Log
+## [1.1.1]
 
+- Install Spectral from the extension
+- Auto update Spectral
```

Comment:
Since it installs Spectral on the machine of the user and not some remote host, I think this is better to ask the user permission to update, or to check for update and install with a prompt. WDYT?

Reply:
i will add a checkbox in the configuration to enable/disable the auto update